etc/adminhtml/system.xml

```diff
@@ -33,19 +33,18 @@
                     <comment>The "Live" mode secret key can be found in Opn Payments Dashboard.</comment>
                 </field>
                 <field id="webhook_status" translate="label comment" type="select" sortOrder="61" showInDefault="1" showInWebsite="1" showInStore="1">
-                    <label>Webhook</label>
+                    <label>Enable webhooks</label>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
-                    <comment>Use webhook.</comment>
+                    <comment>
+                        <![CDATA[Enable webhooks to receive updates when updates to charges and refunds occur. To learn more, see <a href="https://docs.opn.ooo/api-webhooks">our webhooks documentation</a>. No additional configuration necessary.]]>
```

Comment:
```suggestion
                        <![CDATA[Enable webhooks to receive updates when charge and refund events occur. To learn more, see <a href="https://docs.opn.ooo/api-webhooks">our webhooks documentation</a>. No additional configuration necessary.]]>
```

Reply:
Hi, sorry, I'm a bit jetlagged and used the word "updates" twice, so I fixed.

Reply:
No worries, updated.